src/splitInteger.test.js

```diff
@@ -1,17 +1,22 @@
-const splitInteger = require('./splitInteger');
+'use strict';
 
-test(`should split a number into equal parts if a value is divisible by a numberOfParts`, () => {
+const splitInteger = require('./splitInteger');
 
+test(`
+should split a number
+into equal parts if a value is divisible by a numberOfParts`, () => {
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
 });
 
-test(`should return a part equals to a value when splitting into 1 part`, () => {
-
+test(`
+should return a part equals to a value when splitting into 1 part`, () => {
+  expect(splitInteger(8, 1)).toEqual([8]);
 });
```

Comment:
Why did you wrap the test in another test

Reply:
red string with minuses it is what was deleted